src/ert/shared/share/ert/workflows/jobs/internal-gui/scripts/csv_export.py

```diff
@@ -6,6 +6,10 @@
 
 from ert import LibresFacade
 from ert.config import CancelPluginException, ErtPlugin
+from ert.gui.ertwidgets.customdialog import CustomDialog
+from ert.gui.ertwidgets.listeditbox import ListEditBox
+from ert.gui.ertwidgets.models.path_model import PathModel
```

Comment:
I think I moved these in at some point and it was done for the same reason as 
```python
def run_gui_wrapper(args: Namespace, ert_plugin_manager: ErtPluginManager) -> None:
    # Importing ert.gui on-demand saves ~0.5 seconds off `from ert import __main__`
    from ert.gui.main import run_gui  # noqa: PLC0415
```
When you run `ert test_run`, gui will not be imported when you place these imports inside `getArguments`.

Reply:
fixed!